iothub_client/samples/iothub_client_sample_module_sender/start.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
This should probably also be under the "./linux" directory.  Unless we are tying into the "bash on Windows" subsystem stuff for Win10 or assuming a Cygwin or other type Unix'y environment on Win, this looks pretty Linux.

Reply:
Its already in a folder called `docker\linux\amd64`. We should probably make a `Dockerfile` for ARM too. I didn't have the time to setup and test on a Pi.